orders/parser.go

```diff
@@ -0,0 +1,65 @@
+package orders
+
+import (
+	"fmt"
+
+	dip "github.com/zond/godip/common"
+)
+
+type Parser struct {
+	generators []func() dip.Order
```

Comment:
I think you should get rid of the 'generator' concept. Just let each order type provide a single prototype order (i.e. what the current generator returns) and use that instead.